packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -58,11 +71,17 @@ export class FluidCache implements IPersistedCache {
 	private readonly partitionKey: string | null;
 
 	private readonly maxCacheItemAge: number;
+	private readonly immediateClose: boolean;
+	private readonly closeDbAfter: number;
+	private db: IDBPDatabase<FluidCacheDBSchema> | undefined;
+	private dbCloseTimer: ReturnType<typeof setTimeout> | undefined;
 
 	constructor(config: FluidCacheConfig) {
 		this.logger = ChildLogger.create(config.logger);
 		this.partitionKey = config.partitionKey;
 		this.maxCacheItemAge = config.maxCacheItemAge;
+		this.immediateClose = config.immediateClose ?? true;
```

Comment:
Should we try to enable delayed closing by default?

Reply:
@christiango do you have any objection to enable it by default?